packages/cip2/src/types.ts

```diff
@@ -0,0 +1,73 @@
+import { ProtocolParametersAlonzo } from '@cardano-ogmios/schema';
+import { TransactionUnspentOutput, Value, TransactionOutput } from '@emurgo/cardano-serialization-lib-nodejs';
+
+export interface SelectionResult {
+  selection: {
+    /**
+     * A set of inputs, equivalent to a subset of the initial UTxO set.
+     *
+     * From the point of view of a wallet, this represents the value
+     * that has been selected from the wallet in order to cover the total payment value.
+     */
+    inputs: TransactionUnspentOutput[];
```

Comment:
```suggestion
    utxos: TransactionUnspentOutput[];
```

Reply:
The [spec](https://cips.cardano.org/cips/cip2/#results) says "A set of **inputs**, equivalent to a subset of the initial UTxO set.". Please confirm this needs to be renamed.

Reply:
Ok, let's leave it as-is